docs/release_notes/planned_deprecations.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: documentation
+title: Planned Deprecations 
+sidebar: sidebars/documentation.html
+---
+
+# Katello 2.4
+
+ * The Api endpoint /katello/api/v2/repositories/:repository_id/distributions will be removed in favor of attributes within /katello/api/v2/repositories/:repository_id
```

Comment:
What do you think about showing the data attributes so users know what to look for ?